app/controllers/microposts_controller.rb

```diff
@@ -0,0 +1,74 @@
+class MicropostsController < ApplicationController
+  before_action :set_micropost, only: [:show, :edit, :update, :destroy]
```

Comment:
```suggestion
  before_action :set_micropost, only: %i[show, edit, update, destroy]
```

シンボルの配列は%記法を用いることが多いです！
https://qiita.com/mogulla3/items/46bb876391be07921743